releases/v21.2.3.md

```diff
@@ -0,0 +1,150 @@
+
+    ~~~ sql
+    CREATE TABLE t (i INT, j INT, PRIMARY KEY (i), INDEX (j)); INSERT INTO t VALUES (0, 0); WITH   cte1 AS (UPDATE t SET j = 1 WHERE i = 0 RETURNING *),   cte2 AS (UPDATE t SET j = 2 WHERE i = 0 RETURNING *) SELECT 1;
+    ~~~
+
+    Until this is fixed, this change disallows statements with multiple subqueries that modify the same table. Applications can work around this by rewriting problematic statements. For example, the query above can be rewritten as an explicit multi-statement transaction:
+
+    ~~~ sql
+    BEGIN; UPDATE t SET j = 1 WHERE i = 0; UPDATE t SET j = 2 WHERE i = 0; SELECT 1; COMMIT;
+    ~~~
+
+    Or, if it doesn't matter which update "wins", it can be written as multiple non-mutating CTEs on an `UPDATE` statement:
+
+    ~~~ sql
+    WITH   cte1 AS (SELECT 1),   cte2 AS (SELECT 2) UPDATE t SET j = x.j FROM (SELECT * FROM cte1 UNION ALL SELECT * FROM cte2) AS x (j) WHERE i = 0 RETURNING 1;
+    ~~~
+
+    Which in this case could be written more simply as:
+
+    ~~~ sql
+    UPDATE t SET j = x.j FROM (VALUES (1), (2)) AS x (j) WHERE i = 0 RETURNING 1;
+    ~~~
+
+    Note that in these last two rewrites the first update will win, rather than the last. None of these rewrites suffer from the corruption problem. To override this change and allow these statements in spite of the risk of corruption, applications can:
+
+    ~~~sql
+    SET CLUSTER SETTING sql.multiple_modifications_of_table.enabled = true
+    ~~~
+
+    However, with the `sql.multiple_modifications_of_table.enabled` [cluster setting](../v21.2/cluster-settings.html) enabled, there is nothing to prevent this type of corruption from occurring if the same row is modified multiple times by a single statment. To check for corruption, use the `EXPERIMENTAL SCRUB` command:
+
+    ~~~ sql
+    EXPERIMENTAL SCRUB TABLE t WITH OPTIONS INDEX ALL;
+    ~~~
+
+    [#71595][#71595]
+
+- [`RESTORE TABLE`](../v21.2/restore.html) for a regional by row table into a multiregion database with the same regions as the backed up database is now allowed. The user must ensure that the regions in the backed up database and the database being restored into match, and are added in the same order, for the [`RESTORE`](../v21.2/restore.html) to work. [#72088][#72088]
+- The structured payloads used for telemetry logs now include two new fields: `CostEstimate` and `Distribution`. `CostEstimate` is the cost of the query as estimated by the [optimizer](../v21.2/cost-based-optimizer.html), and `Distribution` is the distribution of the DistSQL query plan (local, full, or partial). [#73410][#73410]
+- Fixed a bug which allowed [computed columns](../v21.2/computed-columns.html) to also have [`DEFAULT`](../v21.2/default-value.html) expressions. [#73190][#73190]
+
+### DB Console changes
```

Comment:
I wonder if this heading should be DB Console and Cloud Console. (For the 2nd bullet).